### PR TITLE
Add Calibre

### PR DESCRIPTION
**Pull request by nusr**
add calibre .
It is cross platform e-book manager
## Project URL

https://github.com/kovidgoyal/calibre
## Category

Utilities
## Description

just change **applications.json**
```json
 {
      "short_description" : "cross platform e-book manager.",
      "categories" : [
        "Utilities"
      ],
      "repo_url" : "https://github.com/kovidgoyal/calibre",
      "title" : "calibre",
      "icon_url" : "",
      "screenshots" : [
        "https://github.com/kovidgoyal/calibre/blob/master/resources/images/lt.png"
      ],
      "official_site" : "",
      "languages" : [
        "python"
      ]
    }
```
## Why it should be included to `Awesome macOS open source applications ` (optional)
It's a powerful e-book manager.

## Checklist

- [x] Edit [applications.json](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/applications.json) instead of [README.md](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/README.md).
- [x] Only one project/change is in this pull request
- [x] Screenshots(s) added if any
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

File: applications.json [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+      "categories" : [
+        "Utilities"
+      ],
+      "repo_url" : "https://github.com/kovidgoyal/calibre",
+      "title" : "calibre",
+      "icon_url" : "",
+      "screenshots" : [
+        "https://github.com/kovidgoyal/calibre/blob/master/resources/images/lt.png"
+      ],
+      "official_site" : "",
```
Comment (serhii-londar) [inline applications.json]: Please add "official_site" =https://calibre-ebook.com .

File: applications.json [hunk 2]
```diff
@@ -1,5 +1,21 @@
 {
   "applications" : [
+    {
+      "short_description" : "cross platform e-book manager.",
+      "categories" : [
+        "Utilities"
+      ],
+      "repo_url" : "https://github.com/kovidgoyal/calibre",
+      "title" : "calibre",
+      "icon_url" : "",
```
Comment (serhii-londar) [inline applications.json]: Please add "icon_url" = https://github.com/kovidgoyal/calibre/blob/master/resources/images/lt.png .

File: applications.json (hunk 1 above)
Comment (nusr) [reply applications.json]: do it

File: applications.json (hunk 2 above)
Comment (nusr) [reply applications.json]: I make it.

**Comment by serhii-londar** [comment]
@nusr Thank you! 👍

**Comment by nusr** [comment]
It's my pleasure!

**Pull request merged**